File: katas/8 kyu/reversed-strings/solution.py
# reversed-strings
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def solution(string):
  return string[::-1]
# ------------------------------ CLEVER SOLUTION ----------------------------- #
# ----------------------------------- TEST ----------------------------------- #
test.assert_equals(solution('world'), 'dlrow')
test.assert_equals(solution('hello'), 'olleh')
test.assert_equals(solution(''), '')
test.assert_equals(solution('h'), 'h')

File: katas/8 kyu/reversed-strings/solution.php
<!-- reversed-strings -->
<?php
use PHPUnit\Framework\TestCase;
/* ------------------------------- MY SOLUTION ------------------------------ */
function solution($str) {
  return strrev($str);
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
class ReversedStringsTest extends TestCase {
  public function testExamples() {
    $this->assertSame("dlrow", solution("world"));
    $this->assertSame("olleh", solution("hello"));
    $this->assertSame("", solution(""));
    $this->assertSame('h', solution("h"));
  }
}